lib/rubocop/cop/rspec/verified_double_reference.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for consistent verified double reference style.
+      #
+      # @see https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles
+      #
+      # This cop can be configured in your configuration using the
+      # `EnforcedStyle` option and supports `--auto-gen-config`.
+      #
+      # @example `EnforcedStyle: const`
+      #   # bad
+      #   let(:foo) do
+      #     instance_double('ClassName', method_name: 'returned_value')
+      #   end
+      #
+      #   # good
+      #   let(:foo) do
+      #     instance_double(ClassName, method_name: 'returned_value')
+      #   end
+      #
+      # @example `EnforcedStyle: string`
+      #   # bad
+      #   let(:foo) do
+      #     instance_double(ClassName, method_name: 'returned_value')
+      #   end
+      #
+      #   # good
+      #   let(:foo) do
+      #     instance_double('ClassName', method_name: 'returned_value')
+      #   end
+      class VerifiedDoubleReference < Base
+        include ConfigurableEnforcedStyle
+
+        MSG = 'Prefer `%<style>s` references for verfied doubles.'
+
+        SUPPORTED_STYLES = %w[const string].freeze
+        RESTRICT_ON_SEND = %i[instance_double instance_spy class_double].freeze
```

Comment:
I just now remembered that I have a custom cop similar to this in one of our projects. 😄 

In my version, the list of methods is
```
:mock_model, :stub_model,
:instance_double, :object_double, :class_double,
:instance_spy, :object_spy, :class_spy
```
What do you think? Can we expand the list without problems?

Reply:
@bquorning added